tools/run_tests/performance/prometheus.py

```diff
@@ -0,0 +1,269 @@
+#!/usr/bin/env python3
+
+# Copyright 2022 The gRPC Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# example usage: python3 prometheus.py --url=http://10.108.4.94:9090
+# --pod_type=driver --pod_type=clients --container_name=main
+# --container_name=sidecar
+import argparse
+import json
+import logging
+import requests
+import statistics
+
+from dateutil import parser
+from typing import Any, Dict, List
+
+
+class Prometheus:
+    def __init__(
+        self,
+        url: str,
+        start: str,
+        end: str,
+    ):
+        self.url = url
+        self.start = start
+        self.end = end
+
+    def _fetch_by_query(self, query: str) -> Any:
```

Comment:
Do we know what is the type of resp.json()? I believe something like Dict[str, Any] might be more appropriate.

Reply:
I will try that and update the type.